<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoClass</class>
 <widget class="QMainWindow" name="DemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>200</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_3"/>
   <addaction name="separator"/>
   <addaction name="action_4"/>
   <addaction name="action_5"/>
   <addaction name="action_6"/>
   <addaction name="action_7"/>
   <addaction name="separator"/>
   <addaction name="action_8"/>
   <addaction name="action_10"/>
   <addaction name="action_9"/>
   <addaction name="separator"/>
   <addaction name="action_11"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>新建文档</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>打开文档</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
  </action>
  <action name="action_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
  </action>
  <action name="action_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>级联展开</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>平铺展开</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="demo.qrc">
     <normaloff>:/Demo/exit.svg</normaloff>:/Demo/exit.svg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_11</sender>
   <signal>triggered()</signal>
   <receiver>DemoClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
